rpminspect_runner.sh

```diff
@@ -175,6 +175,12 @@ if [ "${output_format}" == "text" ]; then
     echo "======================================== Test Output ========================================"
 fi
 
+# Update the clamav database
+freshclam
+if [ $? -ne 0 ]; then
+    echo "ERROR: Updating the ClamAV database failed."
```

Comment:
rather a warning? Errors are usually fatal, but we just continue here.

Reply:
The script runs with `set -e`, so it will bail out immediately if `freshclam` commands returns a non-zero status code -- this line won't be executed.

Reply:
Also, we should only update the database if the script was executed without the third optional "$test_name" parameter, or if "$test_name" = "virus" (i.e. not when we run any other inspection).

Reply:
Solid feedback; we can remove the $? check entire if using set -e to validate our outputs.

If we agree this is the proper route to go, I can add the logic around the $test_name - I just wanted to make sure the logic/place makes sense. If so, I can update it. Is the rebase still being worked on?

https://github.com/fedora-ci/rpminspect-image/pull/2#issuecomment-785118233

Reply:
I think the question is what should happen if the update fails. Should we just print warning and continue? Or should the test fail?

I'd probably vote for printing a warning.

> Is the rebase still being worked on?

I dropped the ball there :/ I was actually thinking about ditching bash completely and rewriting the runner script in Python as it is getting more and more complicated: https://github.com/fedora-ci/rpminspect-image/issues/3